docs/en/interacting/monero-wallet-rpc-reference.md

```diff
@@ -349,16 +385,18 @@ Alias:  _None_.
 Inputs:
 
 -   _address_  - string; Public address of the wallet.
--   _message_  - string; (Optional) Should be the same message used in  `get_reserve_proof`.
+-   _message_  - string; If a _message_ was added to `get_reserve_proof` (optional), this message will be required when using `check_reserve_proof`
 -   _signature_  - string; reserve signature to confirm.
 
 Outputs:
 
 -   _good_  - boolean; States if the inputs proves the reserve.
+-   _spent_  - unsigned int; Amount (in [atomic-units](https://www.getmonero.org/resources/moneropedia/atomic-units.html "Atomic Units refer to the smallest fraction of 1 XMR.'")) of the total that has been spent.
+-   _total_  - unsigned int; Total amount (in [atomic-units](https://www.getmonero.org/resources/moneropedia/atomic-units.html "Atomic Units refer to the smallest fraction of 1 XMR.'")) of the reserve that was proven.
```

Comment:
tooltip barely readable. can be modified:
https://github.com/squidfunk/mkdocs-material/discussions/6757#discussioncomment-8431100